src/app/parcels/ParcelsPage.tsx

```diff
@@ -358,6 +359,26 @@ const ParcelsPage: React.FC<{}> = () => {
         }
     }, [parcelId]);
 
+    useEffect(() => {
+        const fetchAndSetClientIdForSelectedParcel = async (): Promise<void> => {
+            const { data, error } = await supabase
+                .from("parcels")
+                .select("client_id")
+                .eq("primary_key", parcelId)
+                .single();
+
+            if (error) {
+                void logError(`Error fetching clientId from database: ${error.message}`);
+                return;
+            }
+
+            const fetchedClientId = data.client_id;
+            setClientIdForSelectedParcel(fetchedClientId);
+        };
+        setClientIdForSelectedParcel(null);
+        fetchAndSetClientIdForSelectedParcel();
```

Comment:
To explicitly indicate that the promise is ignored
```suggestion
        void fetchAndSetClientIdForSelectedParcel();
```

Reply:
Added